apps/site/pages/docs/component-customization/component-customization-overview.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: '  Overview'
+sidebar_label: 'Getting Started'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Component Customization
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+FastStore offers the ability to extend and customize your store components in three ways:
+
+## 1. Content customization through Headless CMS
+
+Change the behavior of specific components by changing their props in the [Headless CMS](/tbd). 
+In the VTEX Admin, go to **Storefront > Headless** and select the section where the desired component is located. 
+From there, you can customize the component by modifying its props. Find the list of available props for customization in the Headless CMS in [Available props for customization in Headless CMS]().
+
+## 2. Theming customization
+Change the store theme and component styling using theming and design tokens. For detailed instructions on customizing through these methods, refer to the [Store Theme]() and [Styling a Component]() documentation.
+
+## 3. Components and sections customization
+
+You have the following customization options for components and sections:
+- [Add extra properties to a native component](/docs/component-customization/adding-extra-properties-to-a-native-component): Modify the behavior of a specific component by passing additional props to it in your store code. This type of customization overrides any changes made in the props through the [Headless CMS](/tbd).
```

Comment:
Actually, we were missing the props overriding feature. So we removed the last paragraph, related to `Overriding or Passing additional props`

I think `Override native's section component's props` suits better than Add extra properties to a native component (I don't think we have the native component concept 🤔 )

Reply:
@hellofanny I agree that this new title suits better. What about `Override component's props`? In the end, the overriding happens in the component's props, right?

Reply:
agreed!